grpc/federation/cel/strings.go

```diff
@@ -0,0 +1,405 @@
+package cel
+
+import (
+	"context"
+	"strings"
+
+	"github.com/google/cel-go/cel"
+	"github.com/google/cel-go/common/types"
+	"github.com/google/cel-go/common/types/ref"
+	"github.com/google/cel-go/common/types/traits"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
+)
+
+const StringsPackageName = "strings"
+
+var _ cel.SingletonLibrary = new(StringsLibrary)
+
+type StringsLibrary struct {
+	typeAdapter types.Adapter
```

Comment:
It seems that typeAdapter is not being used.  Is it possible to remove it?

Reply:
You're right, it's not being used. I've removed it. Thank you for pointing it out.